docker/Dockerfile

```diff
@@ -101,6 +104,7 @@ RUN apt-get -qq update \
     # arch specific packages
     && if [ "${TARGETARCH}" = "amd64" ]; then \
     apt-get -qq install --no-install-recommends --no-install-suggests -y \
+    intel-opencl-icd \
```

Comment:
If this is specifically for openvino then I would recommend putting a comment that explains that so it is clear 👍